bin/env.sh

```diff
@@ -4,7 +4,7 @@ BASEDIR=`dirname $0`/..
 
 if [ ! -d "$BASEDIR/env" ]; then
 
-    virtualenv -q $BASEDIR/env --prompt='(dcos) '
+    virtualenv -p python3 -q $BASEDIR/env --prompt='(dcos) '
```

Comment:
It looks like `venv` is the modern way of creating environment in python3: https://docs.python.org/3/library/venv.html